connectivity/FEATURE_BLE/mbed_lib.json

```diff
@@ -93,10 +93,25 @@
             "value": 15,
             "macro_name": "BLE_GAP_MAX_ADVERTISING_SETS"
         },
+        "ble-gap-host-based-privacy": {
+            "help": "Perform address resolution on the host, not the controller. Controller based privacy is preferred as it happens lower down the stack but this can be used in case controller based privacy is unavailable. If this is enabled the controller will not be used for privacy.",
+            "value": false,
+            "macro_name": "BLE_GAP_HOST_BASED_PRIVACY"
+        },
         "ble-gap-max-advertising-reports-pending-address-resolution": {
             "help": "How many advertising reports can be pending while awaiting private address resolution. This is only used if host privacy is enabled and controller privacy is disabled. Must be non-zero",
             "value": 16,
             "macro_name": "BLE_GAP_MAX_ADVERTISING_REPORTS_PENDING_ADDRESS_RESOLUTION"
+        },
+        "ble-gap-host-privacy-resolving-list-size": {
+            "help": "Used for host privacy. How many pairs of resolvable private addresses and identity address to store.",
+            "value": 8,
+            "macro_name": "BLE_GAP_HOST_PRIVACY_RESOLVED_LIST_SIZE"
```

Comment:
It would made sense for it to be equal to `BLE_SECURITY_DATABASE_MAX_ENTRIES` There won't be more entries than there is in the security DB.

Reply:
replaced with BLE_SECURITY_DATABASE_MAX_ENTRIES